cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc

```diff
@@ -93,8 +97,14 @@ The check to ensure the `iptables` package is installed fails because `iptables`
 |*-*  
 |The https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admincompute/waas/waas_api_discovery#_inspect_discovered_endpoints[API endpoints discovered] on App-embedded deployments are missing workload values and show zero vulnerabilities, although the protected workload has vulnerabilites.
 
+//RLP-104114
```

Comment:
This is not a known issue. This is a change that needs to be on the look ahead.

The phrasing should be similar to:


Addition of New NAT Gateway IP Addresses 

Prisma Cloud is adding new NAT IP addresses for the Compute SaaS Console Region (GCP). The  egress IPs (from console to the internet) in us-east 1 (South Carolina) are:
34.139.64.150
34.139.249.192

Make sure to add these IP addresses to your allow list. 
These IP addresses will be added to the documentation (https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/get-started-with-prisma-cloud/enable-access-prisma-cloud-console#id7cb1c15c-a2fa-4072-b074-063158eeec08_idcb6d3cd4-d1bf-450a-b0ec-41c23a4d4280).